examples/avro-kts/build.gradle.kts

```diff
@@ -17,6 +17,10 @@ schemaRegistry {
                 .addReference("company", "company", -1)
         subject("location-address", "schemas/avro/location-address.avsc", "AVRO")
         subject("location-latlong", "schemas/avro/location-latlong.avsc", "AVRO")
+        subject("my-record","schemas/avro/my-record.avsc", "AVRO")
+            .setMetadata("examples/extra/metadata.json")
+            .setRuleSet("examples/extra/ruleSet.json")
+            .setNormalized(true)
```

Comment:
Based on your comment with confluent cloud, does it work correctly when you run the example directory ?
If not we could comment those line with a short message like you did in the README ?

Reply:
The normalize and metadata work download, ruleset will always return `null` when downloaded, but registering it is fine. I guess the schema registry just ignores it. If you want me to comment it out let me know.

Reply:
If it doesn't fail then it's good to keep it I think 👍